rest/models/quotes_test.go

```diff
@@ -29,6 +30,7 @@ func TestListQuotesParams(t *testing.T) {
 		WithTimestamp(models.LTE, timestamp).
 		WithTimestamp(models.GT, timestamp).
 		WithTimestamp(models.GTE, timestamp).
+		WithDay(year, month, day).
```

Comment:
I don't think we should have this function and a `WithTimestamp(models.EQ, ...)` on the same model here, the `WithDay` shadows that call. In this case they're the same so the expectation doesn't change, but I'm not sure what we're really testing here

Reply:
Makes sense - removed the code.